src/bioregistry/data/bioregistry.json

```diff
@@ -26611,6 +26611,75 @@
       "uri_format": "http://datf.cbi.pku.edu.cn/entry-display.php?id=$1"
     }
   },
+  "dbamp": {
+    "contact": {
+      "email": "leetzongyi@nycu.edu.tw",
+      "name": "Tzong-Yi Lee",
+      "orcid": "0000-0001-8475-7868"
+    },
+    "contributor": {
+      "email": "ji.kang@northeastern.edu",
+      "github": "kanghosaeyo",
+      "name": "Kangho Ji",
+      "orcid": "0009-0009-3402-8013"
+    },
+    "description": "Identifiers represent antimicrobial peptides in the Database of Antimicrobial Peptides (dbAMP) which is an open-access, manually curated database of antimicrobial peptides (AMPs).",
+    "example": "dbAMP_00001",
+    "github_request_issue": 1714,
+    "homepage": "https://awi.cuhk.edu.cn/dbAMP/",
+    "keywords": [
+      "antimicrobial peptides"
+    ],
+    "name": "Database of Anti-Microbial Peptides",
+    "pattern": "^dbAMP_\\d{5}$",
+    "providers": [
+      {
+        "code": "abpdb",
+        "description": "Antimicrobial Peptide Database (ABPDB) is a comprehensive database that aggregates antimicrobial peptide data from DRAMP and dbAMP.",
+        "homepage": "http://www.acdb.plus/ABPDB/",
+        "name": "AntiBacterial Peptides DataBase",
+        "publications": [
+          {
+            "doi": "10.1109/tcbbio.2025.3582844",
+            "pubmed": "40811252",
+            "title": "ABPDB: a Database of Antibacterial Peptides",
+            "year": 2025
+          }
+        ],
+        "uri_format": "http://www.acdb.plus/ABPDB/abp-search.php?keywords=$1"
```

Comment:
http://www.acdb.plus/ABPDB/abp-search.php?keywords=dbAMP_00001 does not work, while http://www.acdb.plus/ABPDB/abp-search.php?keywords=00001 does.